app/views/overview/_service-group.html

```diff
@@ -50,14 +50,16 @@ <h4 class="route-title truncate">
           <!--   <span class="pficon pficon pficon-add-circle-o" aria-hidden="true"></span> -->
           <!-- </div> -->
           <h2>No linked services.</h2>
-          <p>
-            No services are linked to {{service.metadata.name}}.
-            <span ng-if="(services | hashSize) > 1">Add a service to group them together.</span>
-          </p>
-          <div ng-if="(services | hashSize) > 1">
-            <button class="btn btn-primary" ng-show="(services | hashSize) > 1" ng-click="linkService()">
-              Link Service
-            </button>
+          <div ng-if="'services' | canI : 'update'">
```

Comment:
cant decide on this one, what does this look like with just the header vs if you also include the "No services are linked to foo." message?  You might just need hide the button and the "Add a service..." message